core/shortcut_registry.ts

```diff
@@ -219,24 +219,36 @@ export class ShortcutRegistry {
   /**
    * Handles key down events.
    *
+   * - Any `KeyboardShortcut`(s) mapped to the keyscode that cause
+   *   event `e` to be fired will be processed, in order from least-
+   *   to most-recently registered.
+   * - If the shortcut's `preconditionFn` exists it will be called,
+   *   and if it returns false the shortcut will otherwise be ignored
+   *   and processing will continue with the next shortcut, if any.
+   * - The shortcut's `callback` function will the be called.  If it
+   *   returns true, processing will terminate and onKeyDown will
+   *   return true.  If it returns false, processing will continue
```

Comment:
Either use `onKeyDown` for formatting here and in the next bullet, or skip the backticks for all function names in this comment.